src/FeedbackBot/Views/Home/Search.cshtml

```diff
@@ -0,0 +1,66 @@
+<div class="bodyDiv">
+
+    @model IReadOnlyList<FeedbackBot.Controllers.HomeController.issuesContainer>
+    @{
+        ViewData["Title"] = "Home Page";
+    }
+    <a href="/app/@ViewData["AppName"]" style="float:left;">Back</a>
+
+    <h3 class="pageTitle">Search Results For @ViewData["SearchTerm"] in <a href="/app/@ViewData["AppName"]">@ViewData["AppName"]</a></h3>
+    <div class="newWrapper">
+        <div id="newIssueBox">
+            @using (Html.BeginForm("search", "Home",
+                        new { appName = @ViewData["AppName"] }))
+            {
+                <input type="text" id="searchInput" placeholder="Search issues..." name="searchInput" />
+                <input type="hidden" name="appName" value="@ViewData["AppName"]" style="display:none;" />
+                <button type="submit">Submit</button>
+            }
+        </div>
+    </div>
+
+    @foreach (var issue in Model)
```

Comment:
@cszhu is this the same code as on the other page?  If it is, instead of copying you can use a partial in both places